bmq-sdk/src/test/java/com/bloomberg/bmq/it/util/BmqBrokerTestServer.java

```diff
@@ -189,67 +133,23 @@ private void init(int waitTime) throws IOException, InterruptedException {
             np.delete();
         }
 
-        ProcessBuilder pb = null;
-        if (isLocalRun()) {
-            // Since we run multiple tests at the same time starting instances
-            // of BlazingMQ broker in the same location (BMQ_PREFIX),
-            // we need to create a tmp folder for storage, log and stat files
-            tmpFolder = makeTempDir(envPath, "localBMQ_");
-
-            pb =
-                    new ProcessBuilder(
-                            "./bmqbrkr.tsk",
-                            "./bmqbrkr.cfg",
-                            "development",
-                            "set:appConfig/binDir=" + envPath + "/python/bin",
-                            "set:appConfig/etcDir=" + envPath + "/etc",
-                            "set:taskConfig/logController/fileName=" + tmpFolder + "/logs.%T.%p",
-                            "set:appConfig/stats/printer/file=" + tmpFolder + "/stat.%T.%p",
-                            "set:appConfig/plugins/libraries=" + envPath + "/../../plugins",
-                            BMQ_BROKER_LOG_LEVEL.toString());
-
-            final Path storagePath = tmpFolder.resolve("storage");
-
-            Map<String, String> env = pb.environment();
-            env.put("BMQ_PREFIX", envPath);
-            env.put("BMQ_STORAGE", storagePath.toString());
-            logger.info("BlazingMQ Broker storage directory: {}", storagePath);
-
-            outputFile = new File(tmpFolder.resolve("output").toString());
-        } else if (isJenkinsRun()) {
```

Comment:
Jenkins run branch is outdated at all, as some PATH vars in this file, so I removed it